### ISSUE TITLE
Status with eligible operations shows number without operation

### ISSUE THREAD

**Issue by b-butler** [glotzerlab/signac-flow#162]
### Description
When running `python project.py status` the operation eligibility portion shows the number of jobs without a corresponding operation name when no operation is eligible.

### To reproduce

```
import flow
 import signac
 
 project = signac.init_project('EGProject')
 for i in range(3):
     job = project.open_job(dict(a=i))
     job.init()
 
 class Project(flow.FlowProject):
     pass
 
@Project.operation
@Project.post.always
def foo(job):
    pass

if __name__ == '__main__':
    Project().main()
```

`$ python project.py status`

### Error output

![Screen Shot 2019-09-12 at 9 33 51 AM](https://user-images.githubusercontent.com/46760783/64788493-8bb48e80-d540-11e9-9c9a-82fec04672ba.png)

### System configuration

Please complete the following information:

 - Operating System [e.g. macOS]: CentOS 7
 - Version of Python [e.g. 3.7]: 3.6
 - Version of signac [e.g. 1.0]: 1.0.0
 - Version of signac-flow: 8.0

**Comment by csadorf** [comment]
@tcmoore3 That's probably related to the placeholder we needed to introduce, right?

**Comment by tcmoore3** [comment]
Yes, exactly. We added a placeholder with the operation name `’ ‘` to get back the old behavior after implementing jinja for status views, so that’s what shows up in the operation summary. I recommend waiting for the jinja->markdown change for the status view before fixing this. But good catch, @b-butler

**Comment by vyasr** [comment]
@zhou-pj this was resolved in #163, right? If so, feel free to close this issue.

**Comment by zhou-pj** [comment]
yes, thanks for pointing this out!
